instrumentation/instaamqp091/go.mod

```
@@ -1,17 +1,10 @@
 module github.com/instana/go-sensor/instrumentation/instaamqp091
 
-go 1.18
+go 1.16
 
 require (
-	github.com/instana/go-sensor v1.53.0
+	github.com/instana/go-sensor v1.55.0
 	github.com/opentracing/opentracing-go v1.1.0
```

```suggestion
    github.com/opentracing/opentracing-go v1.2.0
```

Query: Don't we have to update the opentracing package here ?